### DISCUSSION TITLE
SMTP server autentication

### DISCUSSION THREAD

**Discussion by AlephArgo** [thalesgroup-cert/Watcher#160] [Ideas]
Hi developers
thanks for your great work.
WOuld it be possible to implement the SMTP server authentication, via account name and password?
In the .env file there are only the two env variables:

    EMAIL_FROM=from@from.com

    SMTP_SERVER=localhost

Thanks
Stefano

**Comment by ygalnezri** [comment]
Hi Stefano,

Thank you for your suggestion and your kind words about the project!

I’m happy to let you know that this feature will be included in **the upcoming release, v2.1**, which is coming very soon. With this update, you’ll be able to configure SMTP server authentication using an account name and password, as well as choose between SMTP and SMTPS.

Stay tuned for the release!

Kind regards,
Ygal

**Reply by ygalnezri** [reply]
Hi!

Thank you for your patience. We are happy to announce that SMTPS has been added in the new Watcher: v2.1 release! You can now configure SMTP authentication and TLS to point to an external email server.

For more details, please check the release notes here: [Watcher v2.1 Release Notes](https://github.com/thalesgroup-cert/Watcher/releases/tag/v2.1) and consult the documentation for configuration details: [Configure your Email notifications](https://thalesgroup-cert.github.io/Watcher/README.html#configure-your-email-notifications) 

Kind regards, 
Ygal
